ngsild-payloads/userProfileDeletionRelatedEntitiesSubscription.jsonld

```diff
@@ -1,5 +1,5 @@
 {
-    "id": "urn:ngsi-ld:Subscription:UserProfileDeletion:RelatedEntitiesDeletionNotification",
+    "id": "urn:ngsi-ld:Subscription:UserProfileDeletion:FluidEntitiesDeletionNotification",
```

Comment:
`FluidEntitiesDeletion` is enough, the corresponding flow does not notify anything